pkg/util/utils.go

```diff
@@ -62,6 +63,63 @@ func ConvertUnsignedBytesToMB(byteSize uint64) int64 {
 	return int64(byteSize / units.MiB)
 }
 
+// ParseMemFree parses the output of the `free -m` command
```

Comment:
Until we have multiple packages using these functions -- can you move it to `pkg/machine`? 

I'd rather this package not exist, to be honest.

Reply:
Will move that too, into a separate PR.

Trying to balance between minimal change so that it can ever get merged, and features so that it actually works...